lexer.go

```diff
@@ -360,9 +360,9 @@ func startWithSlash(s *Scanner) (tok int, pos Pos, lit string) {
 			break
 		}
 		s.r.inc()
-		// in '/*T!', try to consume the 5 to 6 digit version string.
-		commentVersion := s.scanVersionDigits(5, 6)
-		if commentVersion <= CommentCodeCurrentVersion {
+		// in '/*T!', try to match the pattern '/*T![feature1,feature2,...]'.
+		features := s.scanFeatureIDs()
+		if len(features) != 0 && SpecialCommentsController.ContainsAll(features) {
```

Comment:
OK so there's this thing: 

Before this PR, `SELECT /*T! now() */;` is valid because the "version" of `/*T!` without 5 digits following it is always 0. 

So this line should be changed to

```suggestion
		if SpecialCommentsController.ContainsAll(features) {
```